include/tesseract/genericvector.h

```diff
@@ -66,6 +66,10 @@ class GenericVector {
     reserve(size);
     size_used_ = size;
   }
+  void resize_no_init(int size, int rounded_size) {
```

Comment:
I would like to replace class `GenericVector` by standard classes, but each new function makes that more difficult. Try to avoid changing `genericvector.h`.

Reply:
How can I avoid changing GenericVector if I need GenericVector to do something different?

I need to be able to ensure that while a GenericVector of size n still has size n, it has enough space allocated so that I can access elements up to "n rounded up a bit" without causing SEGVs.

If that can be done with the existing GenericVector interface, then please tell me how, and I'll gladly change it.

Reply:
Wouldn't it be sufficient to use the existing API, calling `reserve()` and `truncate()`, for example?

Reply:
It would. I'll move over to that. Nice.